cssTransform.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
Does every transform need to be in a separate file?  Can they be moved to a folder called like `tooling`?  This would declutter the root.